classes/filter_manager.php

```diff
@@ -78,7 +65,7 @@ public function get_all_filters() {
      * @return int points.
      */
     public function get_points_for_event(\core\event\base $event) {
-        foreach ($this->get_all_filters() as $filter) {
```

Comment:
Related to above, I think we can keep this previous method and inject the new `default admin` filters from there.